sympy/ntheory/primetest.py

```diff
@@ -110,6 +110,31 @@ def mr(n, bases):
     return True
 
 
+def isprimeE(n, b):
+    """Test whether n is prime or an Euler pseudoprime to base b."""
```

Comment:
It could be useful to put the definition on a second line

Reply:
Done!